docs/guides/high-availability/operate-site-online

```diff
@@ -0,0 +1,77 @@
+<#import "/templates/guide.adoc" as tmpl>
+<#import "/templates/links.adoc" as links>
+
+<@tmpl.guide
+title="Bring site online"
+summary="This guide describes how to bring a site online so that it can process client requests." >
+
+== When to use this procedure
+
+This procedure describes how to re-add a Keycloak site to the Global Accelerator, after it has previously been taken offline,
+so that it can once again service client requests.
+
+== Procedure
+
+Follow these steps to re-add a Keycloak site to the AWS Global Accelerator so that it can handle client requests.
+
+=== Global Accelerator
+
+. Determine the ARN of the Network Load Balancer (NLB) associated with the site to be brought online
++
+<#include "partials/accelerator/nlb-arn.adoc" />
++
+. Update the Accelerator EndpointGroup to include both sites
+
+<#include "partials/accelerator/endpoint-group.adoc" />
++
+.Output:
+[source,bash]
+----
+{
+    "EndpointGroups": [
+        {
+            "EndpointGroupArn": "arn:aws:globalaccelerator::606671647913:accelerator/d280fc09-3057-4ab6-9330-6cbf1f450748/listener/8769072f/endpoint-group/a30b64ec1700",
+            "EndpointGroupRegion": "eu-west-1",
+            "EndpointDescriptions": [
+                {
+                    "EndpointId": "arn:aws:elasticloadbalancing:eu-west-1:606671647913:loadbalancer/net/a3c75f239541c4a6e9c48cf8d48d602f/5ba333e87019ccf0",
+                    "Weight": 50,
+                    "HealthState": "HEALTHY",
+                    "ClientIPPreservationEnabled": false
+                }
+            ],
+            "TrafficDialPercentage": 100.0,
+            "HealthCheckPort": 443,
+            "HealthCheckProtocol": "TCP",
+            "HealthCheckIntervalSeconds": 30,
+            "ThresholdCount": 3
+        }
+    ]
+}
+----
++
+.. Update the EndpointGroup to include the existing Endpoint and the NLB retrieved in step 1.
++
+.Command:
+[source,bash]
+----
+aws globalaccelerator update-endpoint-group \
+  --endpoint-group-arn arn:aws:globalaccelerator::606671647913:accelerator/d280fc09-3057-4ab6-9330-6cbf1f450748/listener/8769072f/endpoint-group/a30b64ec1700 \
+  --region us-west-2 \
+  --endpoint-configurations '
+  [
+    {
+        "EndpointId": "arn:aws:elasticloadbalancing:eu-west-1:606671647913:loadbalancer/net/a3c75f239541c4a6e9c48cf8d48d602f/5ba333e87019ccf0",
+        "Weight": 50,
```

Comment:
Most of the guides name 128 as the weight, but not here, and also not in the picture. Please double-check.